src/Microsoft.AspNetCore.Mvc/MvcServiceCollectionExtensions.cs

```diff
@@ -16,7 +16,7 @@ public static class MvcServiceCollectionExtensions
         /// Adds MVC services to the specified <see cref="IServiceCollection" />.
```

Comment:
Id rather keep <see cref="IServiceCollection" />. Its way more useful to have the type directly available on the documentation where you can see the type definition on 1 click than having to navigate to the parameter and then to the parameter. Also, this being an extension method, its weird to make reference to the 'this' parameter. Overall, I don't think it makes a lot of difference to use one or the other, it's clear in both cases.